### PR TITLE
Support optional header information in the JSON output

### PR DESCRIPTION
**Pull request by paulwitty**

### CODE REVIEW

File: themes/theme-dark-bronze.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
   "name": "Dark Bronze",
+  "parent": "Dark",
```
Comment (jimmycoyne) [inline themes/theme-dark-bronze.json:3]: let me dig out the names

Comment (jimmycoyne) [reply themes/theme-dark-bronze.json:3]: "name": "MomentumBronzeDark",
  "parent": "BronzeDark",

File: themes/theme-dark-jade.json [hunk 2]
```diff
@@ -1,5 +1,6 @@
 {
   "name": "Dark Jade",
+  "parent": "Dark",
```
Comment (jimmycoyne) [inline themes/theme-dark-jade.json:3]: "name": "MomentumJadeDark",
  "parent": "JadeDark",

File: themes/theme-dark-lavender.json [hunk 3]
```diff
@@ -1,5 +1,6 @@
 {
   "name": "Dark Lavender",
+  "parent": "Dark",
```
Comment (jimmycoyne) [inline themes/theme-dark-lavender.json:3]: "name": "MomentumLavenderDark",
  "parent": "LavenderDark",

File: themes/theme-dark-webex.json [hunk 4]
```diff
@@ -1,5 +1,6 @@
 {
   "name": "Dark Webex",
+  "parent": "Dark",
```
Comment (jimmycoyne) [inline themes/theme-dark-webex.json:3]: "name": "MomentumDark",
  "parent": "Dark",

File: themes/theme-light-bronze.json [hunk 5]
```diff
@@ -1,5 +1,6 @@
 {
   "name": "Light Bronze",
+  "parent": "Light",
```
Comment (jimmycoyne) [inline themes/theme-light-bronze.json:3]: "name": "MomentumBronzeLight",
  "parent": "BronzeLight",

File: themes/theme-light-jade.json [hunk 6]
```diff
@@ -1,5 +1,6 @@
 {
   "name": "Light Jade",
+  "parent": "Light",
```
Comment (jimmycoyne) [inline themes/theme-light-jade.json:3]: "name": "MomentumJadeLight",
  "parent": "JadeLight",

File: themes/theme-light-webex.json [hunk 7]
```diff
@@ -1,5 +1,6 @@
 {
   "name": "Light Webex",
+  "parent": "Light",
```
Comment (jimmycoyne) [inline themes/theme-light-webex.json:3]: "name": "MomentumDefault",
  "parent": "Default",

**Comment by paulwitty** [comment]
Can I suggest that you don't need both the name and parent in the JSON file, if the name is just 'Momentum'+parent?

**Pull request merged**